artifacts/Products/Recommend.recipes

```diff
@@ -19,19 +31,24 @@ particle Recommend in 'source/Recommend.js'
     recommendations `${population}._name_`
 
 particle Chooser in 'source/Chooser.js'
+  in Person person
   in [~a] choices
   inout [~a] resultList
   consume action
     provide set of annotation
       handle choices
-  description `add items from ${choices}`
+  description `add items from ${person.name}'s ${choices}`
 
 particle AlsoOn in 'source/AlsoOn.js'
   in Thing product
   in [Thing] choices
   consume annotation
 
 recipe &addFromWishlist
+  // TODO(sjmiles): chooser won't render if `person.name !== 'Claire'`, but that's
```

Comment:
non-Claire suggestions will appear because `Recommend` particle is always `relevant`. Maybe we need to require that at least one of the UI rendering particles has a `relevance` score (shouldRender returns true). And also this we'll need to tweak the muxer's `shouldRender` method... do you want to file an issue and assign it to me?

in the short term, to avoid bad suggestions, maybe add #claire tag to the handle?

Reply:
Re: relevancy, I infer from your statement that if _any_ particle in a recipe has relevancy, then the recipe is not filtered out of suggestions, is that right? I guess I assumed _all_ particles needed to have relevancy for the recipe to pass. Am I looking at it the wrong way?

Reply:
yes, any particle having relevancy makes the recipe relevant. 
the idea behind this decision was that some particles in the recipe might be optional, but eventually recipes with more particles that have relevancy will be ranked higher than the others. but this is when we really have ranking.